docs/css/styles.css

```diff
@@ -521,11 +523,32 @@ button:-moz-focusring,
       grid-column-end: 4;
       justify-self: end; } }
 
+.ov-masthead--community {
+  background: url("/images/oVirt_Community_Header.png") left top no-repeat var(--ov--color-brand--100); }
+  .ov-masthead--community .ov-heading--lg {
+    max-width: 400px;
+    margin-left: auto;
+    margin-right: auto; }
+
+.ov-layout-styles {
+  margin: var(--ov--spacer--sm);
+  padding: var(--ov--spacer--sm);
+  max-width: 1080px; }
+  @media (min-width: 992px) {
+    .ov-layout-styles {
+      padding: var(--ov--spacer--md); } }
+
+.ov-p-styles {
```

Comment:
you could do this like this
```
.ov-content-styles {
p {
padding: var(--ov--spacer--sm); }
}
ul {
list-style-type: disc;
  padding: var(--ov--spacer--sm);
}
}
```
then you don't need to add a class to every p, ul, etc. As long as the p, ul, etc are wrapped in a `.ov-content-styles`

Reply:
That makes sense to me! I didn't realize you could nest styles like that so that's good to know.